### PR TITLE
Updated to CanJs@5 syntax

### PR DESCRIPTION
**Pull request by Mattchewone**
Updated to use the single `can` import.

### CODE REVIEW

File: hello-world.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+  view: `
+  <div on:click="this.visible = not(this.visible)">
+    {{# if(visible) }}
+      {{message}}
+    {{ else }}
+      Click me
+    {{/ if }}
+  </div>
+  `,
+  ViewModel
```
Comment (justinbmeyer) [inline hello-world.js]: We should probably put the ViewModel inline here.  But not super important.

Comment (Mattchewone) [reply hello-world.js]: I thought about it, it was separated previously. I wasn't sure if it was useful to show it like that. I can change it though, no problem.

Comment (christopherjbaker) [reply hello-world.js]: If it is not inlined, I would suggest moving the `ViewModel` line up higher in the config. When looking back on code, it is very easy to miss a single line that's placed after a large block.

File: hello-world.js [hunk 2]
```diff
@@ -1,19 +1,24 @@
-import Component from "can-component";
-import DefineMap from "can-define/map/map";
-import stache from "can-stache";
+import { Component, DefineMap } from "can";
 
-const HelloWorldVM = DefineMap.extend({
-  visible: { default: false },
-  message: { default: "Hello There!" }
+const ViewModel = DefineMap.extend({
```
Comment (christopherjbaker) [inline hello-world.js]: @justinbmeyer Do we name our ViewModels in examples? I forget.

File: hello-world.js [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
   tag: "hello-world",
-  view: stache("{{#if visible}}{{message}}{{else}}Click me{{/if}}"),
-  ViewModel: HelloWorldVM,
-  events: {
-    click: function() {
-      this.viewModel.visible = !this.viewModel.visible;
-    }
-  }
+  view: `
+  <div on:click="this.visible = not(this.visible)">
```
Comment (christopherjbaker) [inline hello-world.js]: Might be worth giving this block an extra level of indentation so it is visually inside the view line.

**Pull request merged**